autoload/neomake/makers/ft/swift.vim

```diff
@@ -0,0 +1,11 @@
+function! neomake#makers#ft#swift#EnabledMakers()
+    return ['swiftlint']
+endfunction
+
+function! neomake#makers#ft#swift#swiftlint()
+    return {
+        \ 'args': ['lint', '--config', './.swiftlint.yml', '--quiet'],
```

Comment:
Depends on the tool.
Wouldn't that be used by default? (i.e. leave it out here)